agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -802,6 +802,8 @@ bool main_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
                         retry_timer = std::chrono::steady_clock::now() +
                                       std::chrono::seconds(PLATFORM_READ_CONF_RETRY_SEC);
                     }
+                } else {
+                    usleep(PLATFORM_READ_CONF_SLEEP_USEC);
```

Comment:
I considered this approach but wasn't sure if the possible delay of 5 seconds in the shutdown (when performed) is something we wish to avoid.

I will fix this to wait on `PLATFORM_READ_CONF_RETRY_SEC` as suggested and reduce it to 5 seconds.